vhdl_lang/src/syntax/subprogram.rs

```diff
@@ -506,8 +528,14 @@ end function \"+\";
             .subprogram_decl();
         let body = SubprogramBody {
             specification,
-            declarations: vec![],
-            statements: vec![],
+            declarations: WithRegion {
+                item: vec![],
+                range: Position::new(0, 45).range_to(Position::new(1, 0)),
```

Comment:
I have really tried to avoid hard-coding positions in the test cases as it makes them very tedious to update. I have made a lot of utilities to find the position of substrings within the code that I always use.